src/adapter.rs

```diff
@@ -855,6 +887,32 @@ impl<'a> Adapter<'a> for RustdocAdapter<'a> {
                     (ctx, neighbors)
                 }))
             }
+            "Function" | "Method" if matches!(edge_name.as_ref(), "parameters") => {
```

Comment:
```suggestion
            "Function" | "Method" | "FunctionLike" if matches!(edge_name.as_ref(), "parameter") => {
```
Changes:
- Make `parameter` singular -- more detail in another comment below.
- Add `FunctionLike` since `current_type_name` is the statically-determined type bound based on the query, just to prevent a future bug -- also more detail in another comment below.